tools/cl-entry/main.go

```diff
@@ -0,0 +1,128 @@
+package main
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+const (
+	pr = `Must have a Pull Request already open.
+  Enter PR # => `
+
+	kind = `Choose type, one of
+    1. bug
+    2. improvement
+    3. security
+    4. breaking-change
+    5. note
+    6. deprecation
+  Enter Kind => `
+
+	note = `Write a note, for example
+	build: Added make target for creating changelog entries
+  Enter Note => `
+)
+
+var noteRe = regexp.MustCompile(`[a-z0-9/]+: .+`)
```

Comment:
```suggestion
var noteRe = regexp.MustCompile(`[a-z0-9/\s]+: .+`)
```
We have some that use space, like https://github.com/hashicorp/nomad/blob/main/.changelog/14923.txt